src/Server/OutputParser.cs

```diff
@@ -85,6 +101,28 @@ public static string Parse(char[] buffer,int bufferPos, TerminalOptions terminal
                 else token += buffer[i];
             }
 
+            if (isError)
+            {
+                if (!terminalOptions.UseANSI)
+                    return $"ERROR MESSAGE:{AnsiSequences.NewLine}{result}{AnsiSequences.NewLine}";
```

Comment:
nit: maybe just `"ERROR: ..."` and `"WARNING: ..."` to start these?

Reply:
Actually, /Server/ is not the best level of abstraction to enforce opinions about how a particular MUD implementation would like to format errors and warnings anyway.  Plus I wonder if a user could trigger the error/warning formatting in a context that they could trigger colored output; would be really weird.
Combined with the newline problems listed in OutputBuilder and the fact this code author has gone completely dark on multiple levels... I'll be removing from the commit that reconstructs the other pieces of this PR that _do_ work OK.